nestkernel/nest_names.cpp

```diff
@@ -257,6 +259,7 @@ const Name S( "S" );
 const Name scientific( "scientific" );
 const Name screen( "screen" );
 const Name senders( "senders" );
+const Name receivers( "receivers" );
```

Comment:
Do we really want to introduce "receivers"? Wouldn't "targets" be much more in line with terminology we have in NEST otherwise? These are terms the user will see.